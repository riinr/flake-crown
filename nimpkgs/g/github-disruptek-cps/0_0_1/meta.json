{
  "name": "cps",
  "version": "0.0.1",
  "author": "disruptek",
  "desc": "cps",
  "license": "MIT",
  "skipDirs": [],
  "skipFiles": [],
  "skipExt": [],
  "installDirs": [],
  "installFiles": [],
  "installExt": [],
  "requires": [
    {
      "name": "nim",
      "str": ">= 1.0.0",
      "ver": {
        "kind": "verEqLater",
        "ver": "1.0.0"
      }
    },
    {
      "name": "diffoutput",
      "str": "any version",
      "ver": {
        "kind": "verAny"
      }
    }
  ],
  "bin": [],
  "binDir": "",
  "srcDir": "",
  "backend": "c",
  "ref": "refs/tags/0.0.1"
}