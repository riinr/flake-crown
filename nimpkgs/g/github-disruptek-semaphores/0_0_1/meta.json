{
  "name": "semaphores",
  "version": "0.0.1",
  "author": "disruptek",
  "desc": "semaphores",
  "license": "MIT",
  "skipDirs": [],
  "skipFiles": [],
  "skipExt": [],
  "installDirs": [],
  "installFiles": [],
  "installExt": [],
  "requires": [
    {
      "name": "https://github.com/disruptek/balls",
      "str": ">= 3.0.0 & < 4.0.0",
      "ver": {
        "kind": "verIntersect",
        "verILeft": {
          "kind": "verEqLater",
          "ver": "3.0.0"
        },
        "verIRight": {
          "kind": "verEarlier",
          "ver": "4.0.0"
        }
      }
    },
    {
      "name": "https://github.com/disruptek/criterion",
      "str": "< 1.0.0",
      "ver": {
        "kind": "verEarlier",
        "ver": "1.0.0"
      }
    }
  ],
  "bin": [],
  "binDir": "",
  "srcDir": "",
  "backend": "c",
  "ref": "refs/tags/0.0.1"
}