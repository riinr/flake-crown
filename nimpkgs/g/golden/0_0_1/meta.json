{
  "name": "golden",
  "version": "0.0.1",
  "author": "disruptek",
  "desc": "a nim benchmark",
  "license": "MIT",
  "skipDirs": [],
  "skipFiles": [],
  "skipExt": [],
  "installDirs": [],
  "installFiles": [],
  "installExt": [],
  "requires": [
    {
      "name": "nim",
      "str": ">= 0.20.0",
      "ver": {
        "kind": "verEqLater",
        "ver": "0.20.0"
      }
    },
    {
      "name": "cligen",
      "str": ">= 0.9.38",
      "ver": {
        "kind": "verEqLater",
        "ver": "0.9.38"
      }
    }
  ],
  "bin": [],
  "binDir": "",
  "srcDir": "",
  "backend": "c",
  "ref": "refs/tags/0.0.1"
}